app/utils/deep_link/index.ts

```diff
@@ -210,6 +231,14 @@ export const matchPlaybookRunsDeeplink = match<PlaybookRunsPathParams>(PLAYBOOK_
 const PLAYBOOK_RUNS_RETROSPECTIVE = '*serverUrl/playbooks/runs/:playbookRunId/retrospective';
 export const matchPlaybookRunsRetrospectiveDeeplink = match<PlaybookRunsPathParams>(PLAYBOOK_RUNS_RETROSPECTIVE);
 
+type MagicLinkPathParams = {
```

Comment:
just curious, why define the interface in lauch/index.ts and define the type here?

Reply:
If I am not mistaken (a lot of this has been mimicking the behavior around it...) this is just the parameters for the regex match. The ones defined in launch/index.ts are the deeplink objects used later.